tools/image-tag-docker

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+#
+# image-tag-docker runs ./tools/image-tag but sanitizes invalid characters
+# for use in Docker image tags.
+
+TAG=$(sh ./tools/image-tag)
+echo ${TAG//+/-}
```

Comment:
This was being done in a few different places, but also it got complicated to do in the Drone config so I extracted it out to a new script to reduce how many times we have to copy the same sanitization replacement.